kubernetes/linux/setup.sh

```diff
@@ -70,3 +70,5 @@ if [  -e "/var/lib/gems/2.6.0/gems/http_parser.rb-0.6.0/Gemfile.lock" ]; then
       echo "Renaming unused gemfile.lock for http_parser 0.6.0"
       mv /var/lib/gems/2.6.0/gems/http_parser.rb-0.6.0/Gemfile.lock /var/lib/gems/2.6.0/gems/http_parser.rb-0.6.0/renamed_Gemfile_lock.renamed
 fi
+
+wait  # in case telegraf hasn't finished installing yet
```

Comment:
this wait call syncs with telegraf install ? if there is more installed in parallel , this will wait on everything (all children) i assume ?

Reply:
Yes it will wait on all children. Telegraf is the only thing being installed in parallel (since everything else is installed through apt)